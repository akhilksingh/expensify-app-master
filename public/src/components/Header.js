import React from 'react';
import { BrowserRouter, Route ,Switch,Link,NavLink} from 'react-router-dom';
const Header = ()=>(
    <div>
    <h1>Expensify</h1>
    <h2>
    <NavLink to="/" activeClassName="is-active" exact="true">DashBoard </NavLink>
    <NavLink to="/create" activeClassName="is-active">Create Expense</NavLink>
    <NavLink to="/help" activeClassName="is-active"> Help</NavLink>
    </h2>
    </div>
);

export default Header;