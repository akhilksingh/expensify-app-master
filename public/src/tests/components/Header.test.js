import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import {shallow} from 'enzyme';
//import toJSON, { toJson } from 'enzyme-to-json';
import Header from '../../components/Header';

//react-test-renderer

test('should render Header correctly',()=>{
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});

