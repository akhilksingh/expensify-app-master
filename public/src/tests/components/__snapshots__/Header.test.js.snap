// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <h2>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact="true"
      to="/"
    >
      DashBoard 
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/create"
    >
      Create Expense
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/help"
    >
       Help
    </NavLink>
  </h2>
</div>
`;
